src/services/user/user-router.test.ts

```diff
@@ -27,12 +39,24 @@ const OTHER_USER_AUTH = {
     roles: AUTH_ROLE_TO_ROLES[Role.ATTENDEE],
 } satisfies AuthInfo;
 
+const TESTER_EVENT_FOLLOWING = {
+    eventId: "other-event",
+    followers: ["user5", "user8"],
+} satisfies EventFollowers;
+
+const TESTER_ATTENDEE_FOLLOWING = {
+    userId: TESTER.id,
+    events: ["event3", "event9"],
+} satisfies AttendeeFollowing;
+
 // Before each test, initialize database with tester & other users
 beforeEach(async () => {
     Models.initialize();
     await Models.UserInfo.create(TESTER_USER);
     await Models.UserInfo.create(OTHER_USER);
     await Models.AuthInfo.create(OTHER_USER_AUTH);
+    await Models.EventFollowing.create(TESTER_EVENT_FOLLOWING);
```

Comment:
```suggestion
    await Models.EventFollowers.create(TESTER_EVENT_FOLLOWING);
```

Change all instances of eventFollowers to eventFollowing?